.github/workflows/ci.yml

```diff
@@ -54,9 +54,6 @@ jobs:
         run: Scripts/github/prepare-simulators.sh ${{ matrix.platforms }}
       - name: Build and Test Framework
         run: Scripts/build.swift ${{ matrix.platforms }}
-      - name: Upload Coverage Reports
-        if: success()
-        run: Scripts/upload-coverage-reports.sh ${{ matrix.platforms }}
```

Comment:
I was having trouble getting Xcode 11 exporting `lcov` file reports, so I deleted the uploads rather than investing more time here. We don't need coverage on all Xcode runs.